pom.xml

```diff
@@ -250,7 +250,14 @@
                      <exclude>org/apache/parquet/avro/SpecificDataSupplier</exclude> <!-- made public -->
                      <exclude>org/apache/parquet/io/ColumnIOFactory$ColumnIOCreatorVisitor</exclude> <!-- removed non-API class -->
                      <exclude>org/apache/parquet/io/ColumnIOFactory/**</exclude> <!-- removed non-API class and methods-->
-		     <exclude>org/apache/parquet/hadoop/codec/SnappyCompressor</exclude> <!-- added synchronized modifier -->
+                     <exclude>org/apache/parquet/hadoop/codec/SnappyCompressor</exclude> <!-- added synchronized modifier -->
+                     <exclude>org/apache/parquet/bytes/BytesInput</exclude> <!-- moved to parquet-common -->
```

Comment:
We haven't considered moving classes into Jars further up the dependency chain to be breaking so far because you must have parquet-common to use parquet-encoding, and there is no binary incompatibility introduced.

Reply:
My bad. I was worried about moving the CompressionCodecName and CompressionCodecNotSupportedException classes, because I hadn't realized that they were only moved to a different artifact while remaining in the same Java package.